content/collections/audit-works.html

```diff
@@ -1,3 +1,262 @@
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-12
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-10-27
+  </field>
+  <field name="about">
+  </field>
+  <field name="report">
+    /library/audits/sommelier-12
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/PeggyJV/cellar-contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-11
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-08-30
+  </field>
+  <field name="about">
+  </field>
+  <field name="report">
+    /library/audits/sommelier-11
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/PeggyJV/cellar-contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-10
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-08-12
+  </field>
+  <field name="about">
+    The CellarAdaptor was audited again with the added scope of using SavingsDai ERC4626 vault as a position, and no issues were found.
+  </field>
+  <field name="report">
+    /library/audits/sommelier-10
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/PeggyJV/cellar-contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-9
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-08-04
+  </field>
+  <field name="about">
+  </field>
+  <field name="report">
+    /library/audits/sommelier-9
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/PeggyJV/cellar-contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-8
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-06-28
+  </field>
+  <field name="about">
+  </field>
+  <field name="report">
+    /library/audits/sommelier-8
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/PeggyJV/cellar-contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-7
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-04-04
+  </field>
+  <field name="about">
```

Comment:
The Cellar, Registry, CellarFactory, PriceRouter, and SwapRouter contracts were audited, as well as position adaptors integrating Aave, compound, uniswapV3, one inch and 0x.